repository/test-src/org/pentaho/platform/repository2/unified/DefaultUnifiedRepositoryBase.java

```diff
@@ -231,7 +231,7 @@ public void setUp() throws Exception {
     mp.defineInstance( "RepositoryFileProxyFactory", new RepositoryFileProxyFactory( this.jcrTemplate,
         this.repositoryFileDao ) );
     mp.defineInstance( "ITenantedPrincipleNameResolver", new DefaultTenantedPrincipleNameResolver() );
-    mp.defineInstance( "useMultiByteEncoding", Boolean.FALSE );
+    mp.defineInstance( "useMultiByteEncoding", Boolean.TRUE );
     mp.defineInstance( IUnifiedRepository.class, repo );
```

Comment:
Why did we need to change this test ? 
Does that mean we changed system behavior  ?